docs/add-your-app.md

```diff
@@ -0,0 +1,138 @@
+        8080
+      ],
+      "httpPort":8080,
+      "entryPointUrl":"login"
+    }
+  ]
+}
+```
+
+General template fields are pretty straight forward:
+
+<br/>
+
+|property|description|
+|---|---|
+|name|template name|
+|version|template version|
+|description|template additional description|
+|entryPointServiceName|name of the service from the "appServiceTemplates" section that will be used as main entry point for users. it's URL is going to be exposed as the app URL |
+|appServiceTemplates| Array of all services that are part of this app template|
+
+<br/>
+
+For each microservice the app includes add an entry under appServiceTemplates, the app service template format:
+
+<br/>
+
+|property|description|
+|---|---|
+|appServiceName|display name of the service, will be presented in the ocopea app topology|
+|psbType|the orchestrator the app is designated to use - for kubernetes, use k8s|
+|imageName|The full docker image name for running the app|
+|imageType|nodejs|Base docker image type/ technology - used for displaying the app type icon on the app topology view|
+|imageVersion|It is recommended to use a specific docker tag for the image, otherwise use "latest"|
+|environmentVariables|Environment variables expected by the container. it is possible to inject dependency bindings into the variable. In our case we injected the mongoDB server address as part of the environment variable value  
+|dependencies|Array of infrastructure service dependencies of the app service|
+|exposedPorts| Array of ports the app service expects kubernetes to expose externally|
+|httpPort| In case of multiple ports exposed, which port should be used for http access (if applicable). Kubernetes will expose this port externally as port 80 if possible|
+|entryPointUrl|Relative path for entry point url for the service|
+
+<br/>
+
+For each dependency of the app service, add an entry under "dependencies". the dependencies format:
+
+<br/>
+
+|property|description|
+|---|---|
+|type|type of the dependency, used for display only DATABASE,OBJECTSTORE,MESSAGING,CACHE,SCHEDULER,LOGGING,VOLUME,OTHER
+|name|dependency display name|
+|description|Additional description for the dependency and what it is used for|
+|protocols|array supported protocols. This is an array since some apps support alternative protocols (e.g. can use either postgres or mysql according to what is available)|
+
+<br/>
+
+The dependency protocol format:
+
+<br/>
+
+|property|description|
+|---|---|
+|protocolName|name of the protocol - will be matched with DSB supported protocols. e.g. mongodsb,postgres,mysql,s3,docker-volume...|
+|version| optional - use the version field if the service required a minimal version of the protocol
+|conditions| optional - the conditions can be used to restrict protocol attributes for specific protocol requirements 
+
+<br/>
+
+## Generic App Templates
+
+Application templates are built to as generic as possible so the apps could run on as many different clusters as 
```

Comment:
- "built to as" -> "build to be as"
- Add comma after "as possible"
- "so the apps" - > "so that the apps"